.github/workflows/TagBot.yml

```diff
@@ -9,3 +9,4 @@ jobs:
       - uses: JuliaRegistries/TagBot@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          ssh: ${{ secrets.DOCUMENTER_KEY }}
```

Comment:
Why is this needed?

Reply:
Not familiar w/ Documenter but is it not needed for pushing changes?

> https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#travis-ssh

Reply:
It's needed if you're also using a documenter ssh key for another action 
https://github.com/JuliaRegistries/TagBot#setup